### DISCUSSION TITLE
pass custom glob pattern for multiple input files (jsonschema)

### DISCUSSION THREAD

**Discussion by shodhansave** [koxudaxi/datamodel-code-generator#1566] [Q&A]
hello all,

i came across #215 that allows us to pass a directory to `--input`. in my project, there are other, non-json files like readme, changelog, in the folder, that are also picked up and then raise exception as they are not valid json. i checked [here](https://github.com/koxudaxi/datamodel-code-generator/blob/master/datamodel_code_generator/parser/base.py#L502), that we just glob `*` to get all files in the directory. is there a way to pass the glob option so that these non-json files are not picked up?

thank you

**Comment by koxudaxi** [comment]
This issue has already been resolved - non-schema files like README and CHANGELOG are now skipped with warnings instead of causing errors.
